src/sass/components/_tour.scss

```diff
@@ -55,6 +92,7 @@
   box-shadow: 0 0 2px $palette-violet-65;
   color: $palette-violet-65;
   transform: scale(1); // needed to prevent subpixel rounding in Safari
+  z-index: 1;
```

Comment:
Similar to @stacyk's other comment, but I think we could use `$z-index-default` here: https://www.lightningdesignsystem.com/design-tokens/#category-z-index

Reply:
fixed